Ficek_QuantiApp/Ficek_QuantiApp/Domain/Tests/RocketClientTests/RocketClient+happyTests.swift

```diff
@@ -9,72 +9,92 @@ import RequestBuilder
 @testable import RocketClient
 
 final class RocketClientHappyTests: XCTestCase {
-    var cancellables = Set<AnyCancellable>()
+  var cancellables = Set<AnyCancellable>()
+  
+  var testScheduler: TestScheduler<
+    DispatchQueue.SchedulerTimeType,
+    DispatchQueue.SchedulerOptions
+  >! = DispatchQueue.test
```

Comment:
Tohle se vejde na jeden radek ne? Jako naformatovany je to spravne ale je to zbytecny to mit na vice radek, kdyz se to vejde na 120 znaku...